lib/deployinator/templates/layout.mustache

```diff
@@ -456,6 +456,9 @@ _  __  / _  _ \___  __ \__  / _  __ \__  / / /__  / __  __ \_  __ `/_  __/_  __
                                             addStackLockStatus(data.method, data.who, data.lock_time);
                                         }
                                     }
+                                },
+                                error: function (xhrObj, textStatus, errorThrown) {
```

Comment:
All three values are passed to the error function, I'm not sure why they weren't included in the other error callback.. I would rather change that one though since this matches the spec.